partuniverse/templates/pmgmt/part/list.html

```diff
@@ -10,6 +11,12 @@
 
 {% block heading %}{% trans "List of parts" %}{% endblock %}
 
+{% block javascripts %}
```

Comment:
sure, but i just "copied" the implementation of the js inside a template from the add.html template. there it is also in this unclean way done ;)